typings/index.flow.js

```diff
@@ -413,6 +426,7 @@ declare module '@testing-library/react-native' {
 
   declare interface Config {
     asyncUtilTimeout: number;
+    debugOptions?: $Shape<DebugOptions>;
```

Comment:
```suggestion
    defaultDebugOptions?: $Shape<DebugOptions>;
```

Reply:
Oh good catch, I keep missing the flow types updates